specifyweb/frontend/js_src/lib/components/WorkBench/helpers.ts

```diff
@@ -7,13 +7,15 @@ export const downloadDataSet = async (
   name: string,
   rows: RA<RA<string>>,
   columns: RA<string>,
-  delimiter: string
+  delimiter: string,
+  bom: boolean = false
 ): Promise<void> =>
   new Promise((resolve, reject) =>
     stringify(
       [columns, ...rows],
       {
         delimiter,
+        bom,
```

Comment:
(optional) instead of passing the preference to this function, this function could directly access the current preference value by calling `userPreferences.get('queryBuilder','behavior','exportCsvUtf8Bom');`
that would simplify things a bit

same in `doQueryExport`

Reply:
I think I want to be cautious for now and only explicitly enable the BOM through the query export button.
I see downloadDataSet is used in a few other places, mainly for .tsv files, which Excel interprets correctly. It'd be easy to exclude those within the functions still but I think it'd be simpler to move the logic there if we decide to apply the fix universally in a separate issue.